buildbot.mariadb.org/master-galera/master.cfg

```diff
@@ -59,7 +59,7 @@ c['buildbotNetUsageData'] = None
 
 ####### SCHEDULERS
 
-builders_galera=["gal-amd64-ubuntu-1804", "gal-amd64-ubuntu-2004", "gal-amd64-ubuntu-2104", "gal-amd64-ubuntu-2110", "gal-aarch64-ubuntu-1804", "gal-aarch64-ubuntu-2004", "gal-aarch64-ubuntu-2104", "gal-aarch64-ubuntu-2110", "gal-amd64-rhel-7", "gal-amd64-rhel-8", "gal-amd64-debian-9", "gal-x86-debian-9","gal-amd64-debian-10", "gal-amd64-debian-11", "gal-amd64-debian-sid", "gal-x86-debian-sid", "gal-amd64-fedora-34", "gal-amd64-fedora-35", "gal-amd64-centos-7", "gal-amd64-centos-8", "gal-amd64-opensuse-15", "gal-amd64-opensuse-42", "gal-amd64-sles-12", "gal-amd64-sles-15", "gal-ppc64le-debian-9", "gal-ppc64le-debian-10", "gal-ppc64le-debian-11", "gal-ppc64le-debian-sid", "gal-ppc64le-ubuntu-2004", "gal-ppc64le-ubuntu-2104", "gal-ppc64le-ubuntu-2110", "gal-ppc64le-rhel-7", "gal-ppc64le-rhel-8", "gal-ppc64le-centos-7", "gal-aarch64-fedora-34", "gal-aarch64-fedora-35", "gal-aarch64-centos-7", "gal-aarch64-centos-8", "gal-aarch64-debian-10", "gal-aarch64-debian-11", "gal-aarch64-debian-sid", "gal-aarch64-debian-9", "gal-aarch64-rhel-7", "gal-aarch64-rhel-8", "gal-s390x-ubuntu-2004", "gal-s390x-sles-15", "gal-s390x-rhel-8"]
+builders_galera=["gal-amd64-ubuntu-1804", "gal-amd64-ubuntu-2004", "gal-amd64-ubuntu-2104", "gal-amd64-ubuntu-2110", "gal-aarch64-ubuntu-1804", "gal-aarch64-ubuntu-2004", "gal-aarch64-ubuntu-2104", "gal-aarch64-ubuntu-2110", "gal-amd64-rhel-7", "gal-amd64-rhel-8", "gal-amd64-debian-9", "gal-x86-debian-9","gal-amd64-debian-10", "gal-amd64-debian-11", "gal-amd64-debian-sid", "gal-x86-debian-sid", "gal-amd64-fedora-34", "gal-amd64-fedora-35", "gal-amd64-centos-7", "gal-amd64-centos-stream8", "gal-amd64-opensuse-15", "gal-amd64-opensuse-42", "gal-amd64-sles-12", "gal-amd64-sles-15", "gal-ppc64le-debian-9", "gal-ppc64le-debian-10", "gal-ppc64le-debian-11", "gal-ppc64le-debian-sid", "gal-ppc64le-ubuntu-2004", "gal-ppc64le-ubuntu-2104", "gal-ppc64le-ubuntu-2110", "gal-ppc64le-rhel-7", "gal-ppc64le-rhel-8", "gal-ppc64le-centos-7", "gal-aarch64-fedora-34", "gal-aarch64-fedora-35", "gal-aarch64-centos-7", "gal-aarch64-centos-stream8", "gal-aarch64-debian-10", "gal-aarch64-debian-11", "gal-aarch64-debian-sid", "gal-aarch64-debian-9", "gal-aarch64-rhel-7", "gal-aarch64-rhel-8", "gal-s390x-ubuntu-2004", "gal-s390x-sles-15", "gal-s390x-rhel-8"]
```

Comment:
Sorry but it's really not nice to check the diff here!
@vladbogo can't we use another formatting like for https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Reply:
Definitely something that we can do